app/models/department.rb

```diff
@@ -3,7 +3,7 @@ class Department < ApplicationRecord
   
   has_many :users, dependent: :destroy
 
-  validates :department_name, presence: true
+  validates :department_name, presence: true, uniqueness: true
```

Comment:
question: department can be based on organization right? and every organization department can same?